internal/k8s/controller.go

```diff
@@ -2697,6 +2928,30 @@ func getWAFPoliciesForAppProtectLogConf(pols []*conf_v1.Policy, key string) []*c
 	return policies
 }
 
+func getBadosPoliciesForAppProtectDosPolicy(pols []*conf_v1.Policy, key string) []*conf_v1.Policy {
```

Comment:
Could we have  a unit test similar to TestGetWAFPoliciesForAppProtectPolicy?

Reply:
done